payments/models.py

```diff
@@ -296,6 +299,80 @@ def name(self):
     )
 
 
+class TalpaProductAccountingManager(models.Manager):
```

Comment:
FWIW personally I use custom `Queryset`s instead of `Manager`s ~always because they allow method chaining without any actual downsides (that I know of at least :D)

Reply:
Welp, we've been using custom managers throughout the project, so I guess it won't harm to keep this one also for consistency 😅